environments/auth/auth-with-username-no-attributes/amplify/backend/auth/authwithusernamenoat3aaa0954/cli-inputs.json

```diff
@@ -1,53 +1,74 @@
 {
+  "version": "1",
+  "cognitoConfig": {
     "identityPoolName": "authwithusernamenoat3aaa0954_identitypool_3aaa0954",
     "allowUnauthenticatedIdentities": false,
     "resourceNameTruncated": "authwi3aaa0954",
     "userPoolName": "authwithusernamenoat3aaa0954_userpool_3aaa0954",
     "autoVerifiedAttributes": [
-        "email"
+      "email"
     ],
     "mfaConfiguration": "OFF",
     "mfaTypes": [
-        "SMS Text Message"
+      "SMS Text Message"
     ],
     "smsAuthenticationMessage": "Your authentication code is {####}",
     "smsVerificationMessage": "Your verification code is {####}",
     "emailVerificationSubject": "Your verification code",
     "emailVerificationMessage": "Your verification code is {####}",
-    "defaultPasswordPolicy": false,
+    "defaultPasswordPolicy": true,
     "passwordPolicyMinLength": 8,
-    "passwordPolicyCharacters": [],
+    "passwordPolicyCharacters": [
+      "Requires Lowercase",
+      "Requires Uppercase",
+      "Requires Numbers",
+      "Requires Symbols"
+    ],
```

Comment:
FYI, I edited this backend to add strict password policy for the sake of e2e testing. These `environments/` are autogenerated with Amplify CLI.